examples/tools/google/bazeltoolchain/build_simple_bazel_project.rst

```diff
@@ -0,0 +1,135 @@
+.. _examples_tools_bazel_toolchain_build_simple_bazel_project:
+
+Build a simple Bazel project using Conan
+========================================
+
+In this example, we are going to create a string formatter application
```

Comment:
I'd change the "string formatting app" to "a Hello World program" wording to better reflect the simplicity of the built program. Otherwise it seems to me line "string formatting" app will have more contents that might scare readers. What do you think?

Reply:
I agree